tests/e2e/e2e.sh

```diff
@@ -90,10 +46,18 @@ do
 	echo "---" >> "${ISTIO_DIR}"/install/kubernetes/istio-operator.yaml
 done
 
-kubectl get pods --all-namespaces -o wide
+#kind cluster setup
+pushd "${ISTIO_DIR}"
+# shellcheck disable=SC1091
+source "./prow/lib.sh"
+setup_kind_cluster kindest/node:v1.15.3
+popd
 
-pushd "${ISTIO_DIR}" || exit
-  make istioctl
+setup_docker
 
-  HUB=gcr.io/istio-testing TAG=latest E2E_ARGS="--use_operator --test_logs_path=${ARTIFACTS}" make e2e_simple_run
-popd
+KUBECONFIG=$(kind get kubeconfig-path --name="e2e")
+export KUBECONFIG
+
+pushd "${ISTIO_DIR}" || exit
+  HUB=gcr.io/istio-testing TAG=1.5-dev E2E_ARGS="--use_operator --use_local_cluster=true --test_logs_path=${ARTIFACTS}" make e2e_simple_run
```

Comment:
`--use-local-cluster=true` is fine for some conditions (not using metallb) - but I really do want to test load-balancer cases in the 2e test cases locally.  I am good with this minor change (adding metallb into the system) after we get a good baseline).

Reply:
iirc make e2e_simple_run does not build istioctl - which is mandatory for the test case to run - but I could be mistaken (this is why I build it in the original work).